tests/test.py

```diff
@@ -178,18 +178,18 @@ def test_cat(path, expected):
 
 test_cat("/etc/version", b"20221026104022\n")
 
-# assert_value("latest rm1 version", updater.get_latest_version("reMarkable 1"), "3.11.2.5")
-# assert_value("latest rm2 version", updater.get_latest_version("reMarkable 2"), "3.11.2.5")
+# assert_value("latest rm1 version", updater.get_latest_version(HardwareType.RM1), "3.11.2.5")
+# assert_value("latest rm2 version", updater.get_latest_version(HardwareType.RM2), "3.11.2.5")

```

Comment:
```suggestion
assert_value("latest rm1 version", updater.get_latest_version(HardwareType.RM1), "3.20.0.92")
assert_value("latest rm2 version", updater.get_latest_version(HardwareType.RM2), "3.20.0.92")

```
Since you are updating commented  code, we should actually run the test. Looking at this, I as